{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/base.jinja

```diff
@@ -16,6 +16,7 @@
 
         {{ vite_hmr_client() }}
         {{ vite_asset('css/base.js') }}
```

Comment:
what do you think about renaming base.js to styles.js or something to make it clear that's what the purpose is.

Reply:
okay I've renamed it